tests/image_creation.bats

```diff
@@ -45,5 +45,9 @@ function build_image()
 }
 
 @test "Can create euleros image" {
+	if [ "$TRAVIS" = true ]
+	then
+		skip "travis timout, see: https://github.com/kata-containers/osbuilder/issues/46"
```

Comment:
I asked a colleague from our company, we don't have a fast server. It is embarrassing.
I have tried the host server of our isula (a containerOS) community.   It works good.
I'll use that repo URL instead.

Reply:
@liangchenye  I check in travis docuementation, I added `travis_wait` now seems all is OK.